rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -115,6 +115,9 @@ install -m 644 files/securedrop-128x128.png %{buildroot}/usr/share/securedrop/ic
 
 install -m 755 -d %{buildroot}/opt/securedrop
 
+install -m 755 -d %{buildroot}/etc/systemd/logind.conf.d/
+install -m 644 files/10-logind_override.conf %{buildroot}/etc/systemd/logind.conf.d/
+install -m 644 files/logind-override-disable.service %{buildroot}/%{_userunitdir}/
```

Comment:
I think this would be better as a system unit and not user (not sure if a user unit can have a `Before=` a system unit)? the sdw-notify stuff is a system unit (AIUI) because it needs to use the user's GUI session.